### PR TITLE
Adds notes for 4.10.15 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for 4.10.15 release 

Version(s):
Applies to `enterprise-4.10` only

Link to docs preview:

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2892,3 +2892,24 @@ For more information, see xref:../updating/preparing-eus-eus-upgrade.adoc[Prepar
...
+{product-title} release 4.10.15 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:2258[RHBA-2022:2258] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:2257[RHBA-2022:2257] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6960407[{product-title} 4.10.15 container image list]
+
+[id="ocp-4-10-15-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Image Registry Operator blocked installation on installer-provisioned infrastructure (IPI) IBM. With this update, clusters using manual credential minting will now require the administrator role. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2083559[*BZ#2083559*])
```
Comment (sayjadha) [inline release_notes/ocp-4-10-release-notes.adoc:2910]: Suggest revising 'manual credential minting' to 'clusters that mint credentials manually' or something similar for clarity.

Comment (sayjadha) [inline release_notes/ocp-4-10-release-notes.adoc:2910]: Suggest revising 'manual credential minting' to 'clusters that mint credentials manually' or something similar for clarity.

**Review by sayjadha — COMMENTED** [review]
One minor edit, the remaining text LGTM.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -2892,3 +2892,24 @@ For more information, see xref:../updating/preparing-eus-eus-upgrade.adoc[Prepar
...
+{product-title} release 4.10.15 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:2258[RHBA-2022:2258] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:2257[RHBA-2022:2257] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6960407[{product-title} 4.10.15 container image list]
+
+[id="ocp-4-10-15-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Image Registry Operator blocked installation on installer-provisioned infrastructure (IPI) IBM. With this update, clusters that mint credentials manually will now require the administrator role. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2083559[*BZ#2083559*])
```
Comment (maxwelldb) [inline release_notes/ocp-4-10-release-notes.adoc:2910]: Should this be "IBM" or "IBM Cloud"? @opayne1

Comment (maxwelldb) [reply release_notes/ocp-4-10-release-notes.adoc:2910]: Cooouuuuuld also rephrase as something like "Previously, the Image Registry Operator blocked installer-provisioned infrastructure (IPI) installations on IBM Cloud."

**Pull request merged**